plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/Partition.java

```diff
@@ -227,7 +227,7 @@ public void updateNullCount(Map<Integer, Long> nullCounts)
                 this.nullCounts.merge(key, counts, Long::sum));
     }
 
-    public static Map<Integer, Object> toMap(Map<Integer, Type.PrimitiveType> idToTypeMapping, Map<Integer, ByteBuffer> idToMetricMap)
+    public static Map<Integer, Object> convertBounds(Map<Integer, Type.PrimitiveType> idToTypeMapping, Map<Integer, ByteBuffer> idToMetricMap)
```

Comment:
optional: the method doesn't have anything specific to "bounds". May be `convertToValues` or `getValuesFromByteBuffers`, or `convertMetrics`, but don't have a suggestion that sounds obviously better.

Reply:
Keeping current iceberg data structures aside, is there a reason why logically byte buffers would be more suitable for representing bounds, and hence this naming is more appropriate? 

(either way convertBounds reads more easily than toMap, so feel free to resolve this.)

Reply:
>  is there a reason why logically byte buffers would be more suitable for representing bounds

i don't see why we would use different representation in different code paths.
some performance reason?
or some unintentional API difference?
i need to find out.

(also seen here: https://github.com/trinodb/trino/blob/4e30d82cf9490833c9eb8b9881e9c07b1211ef75/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/PartitionTable.java#L293)

> (either way convertBounds reads more easily than toMap, so feel free to resolve this.)

👍